utils/licenses.ts

```diff
@@ -12,48 +12,55 @@ export const licenseDetailsByKind: { [L in LicenseKind]: LicenseDetails<L> } = {
 		short: 'CC-BY',
 		version: '4.0',
 		link: 'https://creativecommons.org/licenses/by/4.0/',
+		spdxIdentifier: 'CC-BY-4.0',
```

Comment:
I decided to add standardized identifiers for licenses for use in the Datacite deposits. https://spdx.dev/